superset/viz.py

```diff
@@ -2062,6 +2062,40 @@ def get_data(self, df):
         }
 
 
+class DeckPolygon(BaseDeckGLViz):
+
+    """deck.gl's Polygon Layer"""
+
+    viz_type = 'deck_polygon'
+    verbose_name = _('Deck.gl - Polygon')
+    deser_map = {
+        'json': json.loads,
+        'polyline': polyline.decode,
+    }
+
+    def get_position(self, d):
```

Comment:
My recent PR got rid of `get_position`, you may need to rebase and remove this method.